### PR TITLE
50 deploy to google cloud

### PR DESCRIPTION
**Pull request by chooseanother**
Guide on how to deploy to google will be written in another document

### CODE REVIEW

File: msdb/compose/production/traefik/traefik.yml
Comment (304125) [inline msdb/compose/production/traefik/traefik.yml:1]: remove

File: msdb/config/settings/base.py [hunk 1]
```diff
@@ -35,6 +37,19 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#locale-paths
 LOCALE_PATHS = [str(BASE_DIR / "locale")]
 
+# Test Gcloud Secret Manager
```
Comment (304125) [inline msdb/config/settings/base.py:40]: remove commented out?

File: msdb/config/settings/base.py [hunk 2]
```diff
@@ -218,7 +233,7 @@
 # Django Admin URL.
 ADMIN_URL = "admin/"
 # https://docs.djangoproject.com/en/dev/ref/settings/#admins
-ADMINS = [("""Kim and Maggie""", "kim-and-maggie@example.com")]
+ADMINS = [("""Kim and Maggie""", "mail@kimdt.tech")]
```
Comment (304125) [inline msdb/config/settings/base.py:236]: can you add two e-mail addresses? fine if not

File: msdb/config/settings/production.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
 SECRET_KEY = env("DJANGO_SECRET_KEY")
 # https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = env.list("DJANGO_ALLOWED_HOSTS", default=["example.com"])
+ALLOWED_HOSTS = env.list("DJANGO_ALLOWED_HOSTS", default=["kimdt.tech"])
```
Comment (304125) [inline msdb/config/settings/production.py:9]: possible to add two emails?

File: msdb/config/settings/production.py [hunk 4]
```diff
@@ -59,7 +65,7 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#default-from-email
 DEFAULT_FROM_EMAIL = env(
     "DJANGO_DEFAULT_FROM_EMAIL",
-    default="Movies and Social DB <noreply@example.com>",
+    default="Movies and Social DB <noreply@kimdt.tech>",
```
Comment (304125) [inline msdb/config/settings/production.py:68]: 2 emails?

**Review by 304125 — CHANGES_REQUESTED** [review]

File: msdb/config/settings/production.py (hunk 4 above)
Comment (chooseanother) [reply msdb/config/settings/production.py:68]: No ):

File: msdb/config/settings/production.py (hunk 3 above)
Comment (chooseanother) [reply msdb/config/settings/production.py:9]: this is allowed host not email

File: msdb/config/settings/base.py (hunk 2 above)
Comment (chooseanother) [reply msdb/config/settings/base.py:236]: Not possible ):

File: msdb/config/settings/base.py (hunk 1 above)
Comment (chooseanother) [reply msdb/config/settings/base.py:40]: Will remove

File: msdb/compose/production/traefik/traefik.yml
Comment (chooseanother) [reply msdb/compose/production/traefik/traefik.yml:1]: :+1:

**Pull request merged**